docs/Documentation/Applications/converge.md

```diff
@@ -0,0 +1,79 @@
+# Running CONVERGE Software
+
+CONVERGE is a computational fluid dynamics (CFD) software package known for its automated meshing, high-fidelity combustion models, and efficiency on complex moving-geometry simulations (like engines, pumps, compressors, injectors, turbines, etc.). For more information about the software's features, see the [CONVERGE
+website](https://convergecfd.com/).
+
+The latest version of CONVERGE installed on Kestrel is version 5.1.0 (default).
+
+## Tips
+
+- **Licensing:**  
+   NREL has one general-use CONVERGE license available. If you expect to run CONVERGE jobs frequently, we recommend obtaining your own license. To do this, please contact ITS to set up a license server for you and provide the port number and server name. 
```

Comment:
Do you have a recommendation for external users wanting to use their own license?